spring-integration-ip/src/main/java/org/springframework/integration/ip/tcp/TcpOutboundGateway.java

```diff
@@ -72,10 +74,9 @@
 
 	private boolean isSingleUse;
 
-	private Expression remoteTimeoutExpression = new LiteralExpression("10000");
+	private Expression remoteTimeoutExpression = new LiteralExpression(String.valueOf(DEFAULT_REMOTE_TIMEOUT));
```

Comment:
There is this one instead:
```suggestion
	private Expression remoteTimeoutExpression = new ValueExpression<>(DEFAULT_REMOTE_TIMEOUT);
```